### PR TITLE
Client can create athletes

### PR DESCRIPTION
**Pull request by holgertidemand**
#### Description
Changes proposed in this pull request:

* adds ability for clients to create athletes 

#### What I have learned working on this feature: 
- To write clean code
- Plan better in design sprint

### CODE REVIEW

File: app/controllers/api/v1/athletes_controller.rb [hunk 1]
```diff
@@ -9,14 +9,31 @@ def show
   end
 
   def update
-    render json: {status: 'Thank you for your vote.'} if @result.updated_votes(params)
+    render json: { status: 'Thank you for your vote.' } if @result.updated_votes(params)
```
Comment (kmthorsnes) [inline app/controllers/api/v1/athletes_controller.rb]: Suggestion for next update: "Thank you for casting your vote!"

File: app/controllers/api/v1/athletes_controller.rb [hunk 2]
```diff
@@ -9,14 +9,31 @@ def show
...
+  end
+
   def find_athlete_and_result
     @athletes = Athlete.all
     @athlete = Athlete.find_by(id: params[:id])
     @result = Result.find_by(athlete: params[:id])
   end
+
+  def render_message(message)
+    render json: { status: message}
```
Comment (kmthorsnes) [inline app/controllers/api/v1/athletes_controller.rb]: space missing?

File: spec/requests/api/v1/create_athlete_spec.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    it 'should return success message if athlete is created' do
+      post '/api/v1/athletes', params: {
+        name: 'Holger Tidemand', age: 26
+      }
+
+      expect(object).to eq 'Please fill in all fields.'
+    end
+  end
+
```
Comment (kmthorsnes) [inline spec/requests/api/v1/create_athlete_spec.rb]: remove whitespace

**Review by kmthorsnes — CHANGES_REQUESTED** [review]

**Review by kmthorsnes — APPROVED** [review]
Looks good!

**Review by davidboll — APPROVED** [review]
LGTM

**Pull request merged**